### ISSUE TITLE
External url

### ISSUE THREAD

**Issue by tech-freak** [demon-rem/kodi-strm#39]
**Is your feature request related to a problem? Please describe.**
Not able to add lite speed server 

**Describe the solution you'd like**
Please add option for external url too for generating strm file
Like xyz.com/2021/video/
Kodi has option for http open index directory but lite speed server is not supported ,with this strm files it will be possible to index and stream data

**Additional context**
Sorry if asking for too much

**Comment by demon-rem** [comment]
Hi,

This repository is a small script I wrote to supplement the existing [Google Drive Add-on](https://kodi.tv/addon/music-add-ons-picture-add-ons-plugins-video-add-ons/google-drive) for Kodi.

As such, this script is ***not really an add-on*** by itself, but instead, helps generate strm files that can then be parsed by the Google Drive Add-on.

If you want the add-on to support more features, or lite speed server, you should instead raise an issue at the [repository ](https://github.com/cguZZman/plugin.googledrive) for the Google Drive Add-on.

**tl;dr**: This repository has nothing to do with Kodi, instead it supplements an existing Kodi add-on. As such, this script can't bypass/overcome the limitations of the add-on. Since the Google Drive add-on does not support strm files outside Google Drive, neither does this script.

Closing this issue since it isn't related to this repository.
